e2e-tests/package.json

```diff
@@ -4,13 +4,32 @@
   "private": true,
   "description": "End-to-end tests for Apex Language Server Extension",
   "scripts": {
-    "test:e2e": "playwright test",
-    "test:debug": "DEBUG_MODE=1 playwright test --debug",
-    "test:visual": "DEBUG_MODE=1 playwright test --ui",
+    "test:e2e": "wireit",
+    "test:debug": "wireit",
+    "test:visual": "wireit",
     "server": "node test-server.js",
     "clean": "echo 'No artifacts to clean in e2e-tests'",
     "clean:all": "npm run clean"
   },
+  "wireit": {
+    "test:e2e": {
+      "command": "playwright test",
+      "output": ["test-results/**", "playwright-report/**"],
+      "files": ["tests/**/*.ts", "utils/**/*.ts", "playwright.config.ts"]
+    },
+    "test:debug": {
+      "command": "DEBUG_MODE=1 playwright test --debug",
+      "output": ["test-results/**"],
+      "files": ["tests/**/*.ts", "utils/**/*.ts", "playwright.config.ts"],
+      "cache": false
+    },
+    "test:visual": {
+      "command": "DEBUG_MODE=1 playwright test --ui",
```

Comment:
strongly recommend using the `env` property from `wireit` instead, here and elsewhere,
...because it works cross-os.